### ISSUE TITLE
Cannot run this module using Terraform 1.0 version

### ISSUE THREAD

**Issue by aburke-incomm** [f5devcentral/terraform-azure-bigip-module#44]
Attempted to run this module and found this error:

```
│ Error: Unsupported Terraform Core version
│ 
│   on .terraform/modules/lle_inf_engage_bigip.bigip/main.tf line 2, in terraform:
│    2:   required_version = "~> 0.13"
│ 
│ Module module.lle_inf_engage_bigip.module.bigip (from
│ github.com/f5devcentral/terraform-azure-bigip-module) does not support
│ Terraform version 1.0.0. To proceed, either choose another supported
│ Terraform version or update this version constraint. Version constraints
│ are normally set for good reason, so updating the constraint may lead to
│ other errors or unexpected behavior.
```

I was able to locally override this (using a comment) once the module was pulled in, but I cannot do this on the deployment agent.

Potential fix:
`required_version = "~> 0.13, <2.0.0"`

**Comment by JeffGiroux** [comment]
Please user terraform version 0.14.x. This module has only been tested with .14 so far. See pre-reqs.

https://github.com/f5devcentral/terraform-azure-bigip-module#prerequisites

## Prerequisites

This module is supported from Terraform 0.13 version onwards.

Below templates are tested and worked in the following version 

Terraform v0.14.0

+ provider registry.terraform.io/hashicorp/azurerm v2.28.0
+ provider registry.terraform.io/hashicorp/null v2.1.2
+ provider registry.terraform.io/hashicorp/random v2.3.0
+ provider registry.terraform.io/hashicorp/template v2.1.2

## Releases and Versioning

This module is supported in the following bigip and terraform version

| BIGIP version | Terraform 1.X  | Terraform 0.14 |
|---------------|----------------|----------------|
| BIG-IP 16.x   |       X        |      X         |
| BIG-IP 15.x   |       X        |      X         |
| BIG-IP 14.x   |       X        |      X         |

You can use tfenv to install and run more than one terraform version on your client if you want to keep 1.x as well as run 0.14. https://github.com/tfutils/tfenv

**Comment by papineni87** [comment]
@aburke-incomm 

Can you pull the latest code, i just fixed the terraform version in the module

```
HYD-ML-00064439:bigip_azure_1nic_deploy papineni$ terraform --version
Terraform v1.0.1
on darwin_amd64
+ provider registry.terraform.io/hashicorp/azurerm v2.71.0
+ provider registry.terraform.io/hashicorp/null v3.1.0
+ provider registry.terraform.io/hashicorp/random v3.1.0
+ provider registry.terraform.io/hashicorp/template v2.2.0

Your version of Terraform is out of date! The latest version
is 1.0.4. You can update by downloading from https://www.terraform.io/downloads.html
HYD-ML-00064439:bigip_azure_1nic_deploy papineni$ 

```

```
HYD-ML-00064439:bigip_azure_1nic_deploy papineni$ terraform apply

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create
 <= read (data resources)

Terraform will perform the following actions:

  # data.azurerm_subnet.mgmt will be read during apply
  # (config refers to values not yet known)
 <= data "azurerm_subnet" "mgmt"  {
      + address_prefix                                 = (known after apply)
      + address_prefixes                               = (known after apply)
      + enforce_private_link_endpoint_network_policies = (known after apply)
      + enforce_private_link_service_network_policies  = (known after apply)
      + id                                             = (known after apply)
      + name                                           = "mgmt-subnet"
      + network_security_group_id                      = (known after apply)
      + resource_group_name                            = (known after apply)
      + route_table_id                                 = (known after apply)
      + service_endpoints                              = (known after apply)
      + virtual_network_name                           = (known after apply)

```

**Comment by papineni87** [comment]
Issue is fixed in v0.9.8, please reopen the issue if you are still facing it.
